_news-items/2024-08-14-tii.html

```diff
@@ -0,0 +1,28 @@
+---
+# Copyright 2024 seL4 Project a Series of LF Projects, LLC.
+# SPDX-License-Identifier: CC-BY-SA-4.0
+title: Welcome TII, Bronze sponsor of the seL4 Summit 2024
+anchor: sponsor24-tii
```

Comment:
You can leave these off for new items, it'll generate one automatically based on the date.

Reply:
These things may need to be put back in. I removed the copyright lines, which resulted in a licence error. Also, I think we do need the title and anchor?

Reply:
Meant just the `anchor` line, sorry. GitHub is showing more than I marked in the review view.